### ISSUE TITLE
No possibility to add repetitive task for few days

### ISSUE THREAD

**Issue by Viktoriia-Chumak** [duy-a/tobidun-issues#7]
**Describe the bug**
It is possible to add tasks only for one day even it is repetitive.

**To Reproduce**
1) Open program
2) Press '+' in Inbox page
3) Add new task and save it

**Expected behavior**
Possibility to chose few days  for repetitive task to avoid every day typing the same information. 

**Smartphone (please complete the following information):**
 - Device: Huawei P20
 - OS: Android 9.0
![Screenshot_20200511_132121_io tobidun app](https://user-images.githubusercontent.com/65408344/82261765-a2ac4b80-992d-11ea-99bd-9841d1d88e3c.jpg)

**Comment by vpeycheva** [comment]
I have a suggestion. You can and a new feature in the task menu (where delete is located), something like 'copy for another date' and/or 'copy for every month'. This can make organizing tasks that have to be done every month on the same date easier. But you'd have to restrict it for like a year or 5 from the current day, to avoid unnecessary storage load.

**Comment by Viktoriia-Chumak** [comment]
Yes, you are right on that way. But it not comfortable for any user at all.
